src/values.rs

```diff
@@ -443,9 +444,9 @@ impl From<Range<f32>> for JitteredValue {
 ///
 /// ```
 /// # use bevy_particle_systems::values::Lerpable;
-/// # use bevy::prelude::Color;
+/// # use bevy::prelude::{Color};
```

Comment:
```suggestion
/// # use bevy::prelude::Color;
```
I don't think that change is necessary, right?